### PR TITLE
feat: Server setup

### PR DESCRIPTION
**Pull request by mixmixmix**
## Description

Various little changes while testing docerized setup for our new server. This version is using archived csv files for oldbahis data.

closes [#issue_number](https://github.com/road86/bahis-infra/issues/20)

### CODE REVIEW

**Review by shaarful — DISMISSED** [review]
It is seems okay to me

File: server-scripts/nightly.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+psql -U postgres -h dash-db -f init.sql
+cd ../input
+# copying local file 
+cp /bahis-backup/coredb_bup.tar.gz .
+tar xf coredb_bup.tar.gz
+psql -U postgres -h dash-db -d coredb -f coredb_bup.sql
+cd /bahis-data/
+python3 server-scripts/import_data.py
+python3 prep_dash/prepgeojson.py
+python3 prep_dash/prep_data.py
```
Comment (sharful09) [inline server-scripts/nightly.sh:12]: add `mkdir -p /bahis-dash/exported_data/ `

**Review by sharful09 — COMMENTED** [review]
Please add  make directory command if not exist

File: server-scripts/nightly.sh (hunk 1 above)
Comment (mixmixmix) [reply server-scripts/nightly.sh:12]: done now

**Review by shaarful — APPROVED** [review]

**Pull request merged**